packages/resolver/test/truffle.js

```diff
@@ -0,0 +1,43 @@
+const assert = require("assert");
+const path = require("path");
+
+// Use transpiled output to properly test the path manipulation
+// truffle resolver utilizes
+const { Truffle } = require("../dist/lib/sources/truffle");
+const resolver = new Truffle({});
+
+describe("truffle resolve [ @win ]", function () {
+
+  describe("assertion contracts", () => {
+    [
+      "Assert",
+      "AssertAddress",
+      "AssertAddressArray",
+      "AssertBalance",
+      "AssertBool",
+      "AssertBytes32",
+      "AssertBytes32Array",
+      "AssertGeneral",
+      "AssertInt",
+      "AssertIntArray",
+      "AssertString",
+      "AssertUint",
+      "AssertUintArray",
+      "SafeSend",
+    ].forEach(lib => {
+      it(`resolves truffle/${lib}.sol`, async () => {
+        const dependency = path.join("truffle", `${lib}.sol`);
+        let result = await resolver.resolve(dependency);
+        assert(
+          result.filePath.includes(dependency),
+          `should have resovled 'truffle${path.sep}${lib}.sol'`
+        );
+      })
+    })
+  });
+
+  describe.skip("DeployedAddresses Contracts", () => {
+    // "DeployedAddresses"
+    // TODO: Not sure how to set this up
```

Comment:
My understanding (based on what @gnidan has said) is that we generally avoid including `TODO` code here... if you're not going to write the code, probably just leave out the whole `describe`.  (Although yes including a note about how it'd be good to add this test would likely be a good thing.)

Reply:
Thanks @haltman-at , I'll remove that and we can get to the test in follow up work

Reply:
OK, again I'd suggest adding back in a comment -- something to the effect of, there should be a `DeployedAddresses` test here but that's more difficult -- but this seems good enough to approve at least. :)